Dockerfile

```diff
@@ -40,6 +40,7 @@ COPY --from=otp-dependencies /build/deps deps
 
 # Install npm dependencies
 COPY assets assets
+RUN npm i --prefix assets
```

Comment:
It is faster to run commands in a Dockerfile file like so:
```dockerfile
RUN command_a && command_b
```

than 

```dockerfile
RUN command_a
RUN command_b
```

because docker backs up the image between each run, so it doesnt have to rebuild everything at every build. plz fix